api/envoy/api/v2/core/config_source.proto

```diff
@@ -96,6 +100,22 @@ message RateLimitSettings {
   google.protobuf.DoubleValue fill_rate = 2 [(validate.rules).double = {gt: 0.0}];
 }
 
+// Configuration defining a jittered exponential back off strategy.
+message BackOffStrategy {
```

Comment:
Can you move this message to a common file? I would like to refer it in the remote data source.

Reply:
Yes. I will pick this back up again after New Year's.